src/Okta.Sdk/Client/Configuration.cs

```diff
@@ -728,7 +758,7 @@ public static string ToDebugReport()
             report += "    OS: " + System.Environment.OSVersion + "\n";
             report += "    .NET Framework Version: " + System.Environment.Version  + "\n";
             report += "    Version of the API: 3.0.0\n";
-            report += "    SDK Package Version: 6.0.7\n";
+            report += "    SDK Package Version: 6.0.8\n";
```

Comment:
The version should probably be retrieved from the assembly to make sure we don't accidently report the wrong version if this line is not updated in the future, see [Version Class](https://learn.microsoft.com/en-us/dotnet/api/system.version?view=net-8.0#remarks):

```
Version version = Assembly.GetExecutingAssembly().GetName().Version;
string version  = $"{version.Major}.{version.Minor}.{version.Build}";
```

Reply:
This is all taken care of by the codegen, you update the config.json, and the generator does the work for us.